app/Http/Requests/SplitRequest.php

```diff
@@ -24,6 +24,7 @@ public function rules()
         return [
             'competition_end_date' => 'required|date|after:today',
             'leaderboard_msg_day' => 'required',
+            'rules' => 'required',
```

Comment:
I think we could add this should be a url as well https://laravel.com/docs/5.2/validation#rule-url